src/graph.cc

```diff
@@ -27,8 +27,12 @@
 #include "state.h"
 #include "util.h"
 
-bool Node::Stat(DiskInterface* disk_interface, string* err) {
-  return (mtime_ = disk_interface->Stat(path_, err)) != -1;
+bool Node::Stat(DiskInterface* disk_interface,
+                DiskInterface::SymlinkTreatment symlink_treatment,
+                string* err) {
+  METRIC_RECORD("node stat");
```

Comment:
Is this a mismerge? danalbert moved this METRIC_RECORD into disk_interface recently.